### ISSUE TITLE
Datepicker dialog does not render properly on Android 8.0

### ISSUE THREAD

**Issue by venkateshsj** [davecoffin/nativescript-modal-datetimepicker#15]
### If the demo apps cannot help and there is no issue for your problem, tell us about it
Please, ensure your title is less than 63 characters long and starts with a capital
letter.

### Which platform(s) does your issue occur on?
`Android 8.0 (OnePlus3 - OxygenOS 5.0.2`

### Please, provide the following version numbers that your issue occurs with:

- CLI: (run `tns --version` to fetch it)
- Cross-platform modules: (check the 'version' attribute in the
`node_modules/tns-core-modules/package.json` file in your project)
- Runtime(s): (look for the `"tns-android"` and `"tns-ios"` properties in the `package.json` file of your project)
- Plugin(s): (look for the version numbers in the `package.json` file of your
project and paste your dependencies and devDependencies here)

```
nativescript - 3.4.3 - Up to date
tns-core-modules - 3.4.1 - Up to date
tns-android - 3.4.2 - Up to date
tns-ios - 3.4.1 - Not installed
```

```
Plugins:
nativescript-modal-datetimepicker - v1.1.1
```

### Please, tell us how to recreate the issue in as much detail as possible. 
Describe the steps to reproduce it.

```
1. Simply create a new javascript project
2. Add nativescript-modal-datetimepicker plugin
3. Add code to view picker on button click
```

You will see that the header is offset to the left showing white background on right side. Refer the following screenshot,
![screenshot_20180405-122757_01](https://user-images.githubusercontent.com/7588513/38351587-c3395b26-38cd-11e8-83d5-7f507a6ced38.jpg)

I could not find any way to rectify this issue, hence reporting it to you. Also, please please provide code to style the datepicker/timepicker using styles.xml or even programmatically.

I tried adding following code to styles.xml inside values-21 folder, but no effect,
```
<style name="SpinnerDatePicker" parent="android:Widget.Material.Light.DatePicker">
        <item name="android:datePickerMode">calendar</item>
        <item name="android:background">@color/android:white</item>
        <item name="android:textColorPrimaryInverse">@color/ns_accent</item>
        <item name="android:textColorPrimary">@color/ns_accent</item>
        <item name="android:textColorSecondary">@color/ns_accent</item>
        <item name="android:textColorSecondaryInverse">@color/ns_accent</item>
        <item name="android:textColorTertiary">@color/ns_accent</item>
        <item name="android:headerBackground">@color/android:white</item>
        <item name="android:yearListSelectorColor">#0000ff</item>
        <item name="android:dayOfWeekBackground">#ff5555</item>
    </style>
```

### Is there any code involved? 
 - provide a code example to recreate the problem 
 - (EVEN BETTER) provide a .zip with application or refer to a repository with application where the problem is reproducible.

**Comment by atchakraborty** [comment]
Same thing is happening for me as well

**Comment by Ericky14** [comment]
Please help with this issue, the TimePicker styles properly, but the DatePicker always has the green background.

**Comment by shiv19** [comment]
@venkateshsj
Which is the `targetSdk` version that you are compiling your app for?

**Comment by Ericky14** [comment]
@shiv19 Mine's 26 if that helps..

**Comment by pattrawuth2530** [comment]
Have a same problem.
Run on android emulator Nexus9 API 27 (Android 8.1).
"nativescript-modal-datetimepicker": "^1.1.7",

**Comment by shiv19** [comment]
@pattrawuth2530 yes this is a known issue currently

**Comment by sarawieckowska** [comment]
Any update on this issue?

**Comment by shiv19** [comment]
@sarawieckowska
No update yet.
Help Wanted.

**Comment by MihkelBaranov** [comment]
Temporary workaround could be to change the themeId to 0 and use the parent context's default theming

https://github.com/davecoffin/nativescript-modal-datetimepicker/blob/c7a1909f1c60c772680b0c85593713a895c02f15/src/modal-datetimepicker.android.ts#L28

![screenshot_20181219-120947](https://user-images.githubusercontent.com/3790782/50213752-2eabf380-0387-11e9-8866-6c114629022c.jpg)

for reference 
https://developer.android.com/reference/android/app/DatePickerDialog#DatePickerDialog(android.content.Context,%20int)

**Comment by shiv19** [comment]
@MihkelBaranov!
Thank you for this :)

**Comment by shiv19** [comment]
@MihkelBaranov
Released as 1.1.12
